src/java.desktop/macosx/classes/sun/java2d/MacosxSurfaceManagerFactory.java

```diff
@@ -51,7 +51,7 @@
     public VolatileSurfaceManager createVolatileManager(SunVolatileImage vImg,
                                                         Object context)
     {
-        return MacOSFlags.isMetalEnabled() ? new MTLVolatileSurfaceManager(vImg, context) :
+        return CGraphicsDevice.useMetalPipeline() ? new MTLVolatileSurfaceManager(vImg, context) :
```

Comment:
similar to my other naming comments this is more "using" than "use"  - or enabled again.
I'd like consistent naming where it makes sense.
But also this looks like another case where I'd prefer to see us have a pattern that retrieves  the installed pipeline.
So this entire method body is something like 
getInstalledPipeline().createVolatileSurfaceManager(vImg, context);